include/libopencm3/pac55xx/ccs.h

```diff
@@ -65,6 +143,8 @@
 #define CCS_PFMUXSELR CCS_MUXSELR(CCS_PORTF)
 #define CCS_PGMUXSELR CCS_MUXSELR(CCS_PORTG)
 #define CCS_MUXSELR_MASK 0x7
+#define CCS_MUXSELR_MASK_PIN(pin) (CCS_MUXSELR_MASK << ((pin)) * 4)
```

Comment:
I'm just about to merge this, and it complains on this as a conflict with 89e90e0e5da5258432f4e16fdcf5911d3f8a22b2
I _believe_ from inspection that _that_ commit is correct, and this has the bracketing about pin wrong, but please let me know if I've got that wrong! :)

Reply:
Yes, you are correct. the other one is right, while this one is wrong.